iib/workers/tasks/build_merge_index_image.py

```diff
@@ -177,6 +177,7 @@ def handle_merge_request(
     target_index=None,
     overwrite_target_index=False,
     overwrite_target_index_token=None,
+    distribution_scope=None,
```

Comment:
Will there be a future PR for passing the `distribution_scope` to this task from `api_v1.py`?

Reply:
I had the same question.